### DISCUSSION TITLE
Is it possible to pass an argument to a test?

### DISCUSSION THREAD

**Discussion by stephenmday** [cypress-io/cypress#24194] [Questions and Help] [answered]
Along the lines of this article - https://blog.bitsrc.io/how-to-pass-command-line-arguments-in-node-js-46af7960b45f
I would like to run a test in Jenkins and add parameterized values to the test run, but do not know how to get those values into the cy.js test.

**Comment by mkonev-casafari** [comment, marked as answer]
Hi,

you can pass variables in following way 

when run cypress
```
npx cypress run --env yourVariable=value
```

and use it in test
```
const var = Cypress.env('yourVariable');
```

for more details read following article https://docs.cypress.io/guides/guides/environment-variables#Setting

**Reply by stephenmday** [reply]
Thank you, I was able to get it working
